spynnaker/pyNN/models/neural_projections/connectors/from_list_connector.py

```diff
@@ -47,53 +46,6 @@ def __init__(self, conn_list, safe=True, verbose=False):
         self._delays = None
         self._converted_weights_and_delays = False
 
-    @staticmethod
-    def _split_conn_list(conn_list, column_names):
-        """ takes the conn list and separates them into the blocks needed
```

Comment:
Taking out this function leads to an error as it is called from the sPyNNaker8 from_list_connector; it might be best to move this function up there rather than keeping it here... again it depends somewhat on how much PyNN 0.7 support we want to bother with.